Makefile

```diff
@@ -49,22 +49,25 @@ transfer:
   		--network celestia-zkevm-hl-testnet_celestia-zkevm-net \
   		--volume celestia-zkevm-hl-testnet_celestia-app:/home/celestia/.celestia-app \
   		ghcr.io/celestiaorg/celestia-app-standalone:feature-zk-execution-ism \
-  		tx warp transfer 0x726f757465725f61707000000000000000000000000000010000000000000000 1234 0x000000000000000000000000d7958B336f0019081Ad2279B2B7B7c3f744Bce0a "1000" \
+  		tx warp transfer 0x726f757465725f61707000000000000000000000000000010000000000000000 1234 0x000000000000000000000000aF9053bB6c4346381C77C2FeD279B17ABAfCDf4d "1000" \
   		--from default --fees 400utia --max-hyperlane-fee 100utia --node http://celestia-validator:26657 --yes
 .PHONY: transfer
 
 ## transfer-back: Transfer tokens back from the EVM roll-up to celestia-app.
-# TODO: https://github.com/celestiaorg/celestia-zkevm-hl-testnet/issues/34
 transfer-back:
 	@echo "--> Transferring tokens back from the EVM roll-up to celestia-app"
-	@go run ./testing/demo/pkg/transfer/ transfer-back
+	@cast send 0xa7578551baE89a96C3365b93493AD2D4EBcbAe97 \
+  		"transferRemote(uint32, bytes32, uint256)(bytes32)" \
+  		69420 0000000000000000000000006A809B36CAF0D46A935EE76835065EC5A8B3CEA7 1000 \
```

Comment:
This is the recipient on celestia-app (i.e. the `default` account added in the celestia-app state initialisation).

This is not `0x` prefixed and is the hex address left padded to ensure it meets the requirement of 32 bytes. This is gotten easily by running:

```
celestia-appd debug addr celestia1d2qfkdk27r2x4y67ua5r2pj7ck5t8n4890x9wy
Address: [106 128 155 54 202 240 212 106 147 94 231 104 53 6 94 197 168 179 206 167]
Address (hex): 6A809B36CAF0D46A935EE76835065EC5A8B3CEA7
Bech32 Acc: celestia1d2qfkdk27r2x4y67ua5r2pj7ck5t8n4890x9wy
Bech32 Val: celestiavaloper1d2qfkdk27r2x4y67ua5r2pj7ck5t8n48qsyucz
Bech32 Con: celestiavalcons1d2qfkdk27r2x4y67ua5r2pj7ck5t8n485rhq5r
```